src/detections/hooks/use-content.jsx

```diff
@@ -0,0 +1,9 @@
+import { BarChart } from 'detections/chart/bar'
+import { Error } from 'detections/chart/error'
+import { Loading } from 'detections/chart/loading'
+
+export const useContent = () => ({
```

Comment:
Let's change this file's extension to `.js`.

Reply:
done